dash/backend/src/email-templates/error-email.hbs

```diff
@@ -30,7 +30,7 @@
         </p>
 
         <p>
-            {{{data}}}
+            {{data}}
```

Comment:
Did you change from using something besides handlebars?

Handlebars does use triple brackets

Reply:
Good catch!